Resources/ServerInfo/Guidebook/Antagonist/Nuclear Operatives.xml

```diff
@@ -0,0 +1,87 @@
+
+If you hear this message on round start then congratulations! You have just been chosen to be a nuclear operative for the syndicate. You have one goal, to blow up the space station with a nuclear fission explosive.
+
+## Operatives
+
+Your team consists of three to five operatives, two of which have a special role:
+- The [color=#a4885c]Nukie Commander[/color] wears the unique commander’s hardsuit, and will create or approve of the battle plan.
+<Box>
+<GuideEntityEmbed Entity="ClothingOuterHardsuitSyndieCommander" Caption="Commander Hardsuit"/>
+</Box>
+
+- The [color=#a4885c]Nukie Agent[/color] wears the black elite hardsuit, and can act as a combat medic for the mission.
+<Box>
+<GuideEntityEmbed Entity="ClothingOuterHardsuitSyndieElite" Caption="Agent Hardsuit"/>
+</Box> 
+
+- There are one to three [color=#a4885c]Nuclear Operatives[/color], who have the normal blood red hardsuit.
+<Box>
+<GuideEntityEmbed Entity="ClothingOuterHardsuitSyndie" Caption="Blood Red Hardsuit"/>
+</Box>
+
+If you have been selected for either of these special roles but don't feel ready, one of your teammates can take responsibility.
+
+## Preparation
+
+Taking on a whole station in a fight is a difficult job, luckily you are well prepared for the job. Every operative has an [color=#a4885c]Uplink[/color]. It comes with 40 [color=#a4885c]Telecrystals[/color], which you can use to purchase gear. What you get depends on the strategy picked by your commander or voted for by the team. Only your commander and agent will have passenger access, but no external airlock access. This means you will need something to hack or blast your way onto the station.  In general you should need some weapons, something to breach doors, and utility/healing.
+
+<Box>
+<GuideEntityEmbed Entity="BaseUplinkRadio40TC" Caption="The Uplink"/>
+</Box>
+
+## Getting to the Station
+
+After gearing up and creating a plan, grab a jetpack from your armory and go with your other operatives to the [color=#a4885c]Nukie Shuttle[/color]. Here you will find some extra explosives and tools, as well as your nuke and nuke codes.
+You will find an [color=#a4885c]IFF Console[/color] on the shuttle, it allows you to hide from other ships and mass scanners. Make sure that [color=#a4885c]Show IFF[/color] and [color=#a4885c]Show Vessel[/color] are toggled off to hide your shuttle from the crew. When everyone is ready, FTL to the station and fly to it with a jetpack. Don't forget the nuclear fission explosive on your ship if you are going to use it, and definitely don't forget the nuke codes or pinpointer.
+
+## The Disk
+
+You will notice that each operative starts with a [color=#a4885c]Pinpointer[/color]. This device is one of the most important to the mission, and you should keep it with you at all times. Turn on the pinpointer when you arrive to the station and it will always point to the [color=#a4885c]Nuke Disk[/color], your next objective.
```

Comment:
```suggestion
You will notice that each operative starts with a [color=#a4885c]Pinpointer[/color]. This device is one of the most important items to the mission, and you should keep it with you at all times. Turn on the pinpointer when you arrive at the station and it will always point to the [color=#a4885c]Nuke Disk[/color], your next objective.
```